Apps/WebClient/src/ClientApp/src/components/timeline/encounter.vue

```diff
@@ -0,0 +1,194 @@
+                <b-row class="justify-content-between">
+                    <b-col cols="auto">
+                        <strong>{{ entry.practitionerName }}</strong>
+                    </b-col>
+                </b-row>
+            </b-col>
+        </b-row>
+        <b-row class="my-2">
+            <b-col class="leftPane"></b-col>
+            <b-col>
+                <b-row>
+                    <b-col cols="auto">
+                        <strong>
+                            Specialty Description:
+                        </strong>
+                        {{ entry.specialtyDescription }}
+                    </b-col>
+                </b-row>
+                <b-row>
+                    <b-col>
+                        <div class="d-flex flex-row-reverse">
+                            <b-btn
+                                v-b-toggle="
+                                    'entryDetails-' + index + '-' + datekey
+                                "
+                                variant="link"
+                                class="detailsButton"
+                                @click="toggleDetails()"
+                            >
+                                <span class="when-opened">
+                                    <font-awesome-icon
+                                        icon="chevron-up"
+                                        aria-hidden="true"
+                                    ></font-awesome-icon
+                                ></span>
+                                <span class="when-closed">
+                                    <font-awesome-icon
+                                        icon="chevron-down"
+                                        aria-hidden="true"
+                                    ></font-awesome-icon
+                                ></span>
+                                <span v-if="detailsVisible">Hide Details</span>
+                                <span v-else>View Details</span>
+                            </b-btn>
+                        </div>
+                        <b-collapse
+                            :id="'entryDetails-' + index + '-' + datekey"
+                            v-model="detailsVisible"
+                        >
+                            <div>
+                                <div class="detailSection">
+                                    <div>
+                                        <strong>Location:</strong>
+                                    </div>
+                                    <div>
+                                        {{ entry.clinic.name }}
+                                    </div>
+                                    <div>
+                                        {{ entry.clinic.address }}
+                                    </div>
+                                    <div>
+                                        {{
+                                            phoneUtil.formatPhoneNumber(
```

Comment:
as we use phoneUtil here, should we remove the private formatPhoneNumber below?

Reply:
made further changes - see latest